Neos.Neos/Resources/Private/Fusion/Prototypes/Image.fusion

```diff
@@ -0,0 +1,5 @@
+prototype(Neos.Neos:Image) < prototype(Neos.Media:Image)
+# ImageUri object works exactly the same way as the uri.image ViewHelper in the Neos.Media package
+prototype(Neos.Neos:ImageUri) < prototype(Neos.Media:ImageUri)
+# ImageTag object works exactly the same way as image ViewHelper in the Neos.Media package
+prototype(Neos.Neos:ImageTag) < prototype(Neos.Media:ImageTag)
```

Comment:
I think those two prototypes should be declared as deprecated. 

It would be cool to have a code migration that replaces those. Since this is a clear 1:1 mapping that should be easy and will allow us to safely remove them in Neos 6 or 7.